### DISCUSSION TITLE
How dangerous is it to expose my server URL to the public?

### DISCUSSION THREAD

**Discussion by TechStudent10** [pocketbase/pocketbase#1397] [Q&A] [answered]
Hi everyone!

I'm working on an open-source app that requires Pocketbase. I've been wondering how I should implement the server URL.

My first thought was to just leave the server URL in plain text. But then I began thinking about what would happen if someone just had access to my db.

Would exposing the URL allow some malicious actor to modify my DB or am I worrying about nothing?

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure that I understand what do you mean by "_leave the server URL in plain text_".

Access to your data depends on your collections API Rules (by default only admins have permissions).

It is OK the PocketBase server address to be public. In case an API rule forbid access to a certain collection, an unauthorized request client will get a 40* error when attempting to access it.

**Reply by TechStudent10** [reply]
Okay thanks!

> I'm not sure that I understand what do you mean by "_leave the server URL in plain text_".

By that I mean leaving the URL in the file, as the code is open source.
